examples/README.md

```diff
@@ -194,30 +194,38 @@ Sources [ScalaRiakParquetExample.scala](./src/main/scala/com/basho/riak/spark/ex
 
 The Spark-Riak Connector can be used with Spark Streaming. To demonstrate this usage, we will work through two small Scala examples, one for Riak KV and the other for Riak TS.
 
-These examples require the use of Kafka. Please install Kafka and setup a Kafka broker prior to running this example. We will assume that there is a Kafka broker running at `127.0.0.1:9092` with a topic called `streaming`. Instructions for setting up Kafka topics can be found in [this guide](https://kafka.apache.org/documentation.html#quickstart). You can create a broker and topic with the following:
+These examples require the use of Kafka. Please install Kafka and setup a Kafka broker prior to running this example. We will assume that there is a Kafka broker running at `127.0.0.1:9092`. Instructions for setting up Kafka topics can be found in [this guide](https://kafka.apache.org/documentation.html#quickstart). You can create a broker with the following:
 
 ```
 path/to/kafka/bin/zookeeper-server-start.sh config/zookeeper.properties
 path/to/kafka/bin/kafka-server-start.sh config/server.properties
-path/to/kafka/bin/kafka-topics.sh --create --zookeeper localhost:2181 --replication-factor 1 --partitions 1 --topic streaming
 ```
 
 We also assume Riak KV/TS is installed and there is a Riak KV/TS node running at `127.0.0.1:8087`. You can find instruction to do so [here](http://docs.basho.com/riak/ts/latest/installing/). 
 
 Riak KV, Kafka and Spark master hostnames must be specified in [config.sh](./src/main/repl/conf/config.sh) prior to running the examples.
 
+**Important Note**: [config.sh](./src/main/repl/conf/config.sh) will attempt to gather your local scala version. This version number will be used in [run-example](./src/main/repl/bin/run-example) to pull the appropriate `spark-streaming-kafka` and `kafka` libraries from spark-packages.org. This can cause an error if your local Spark and the Spark-Riak-Connector were built for a different Scala version than your local Scala. For example, Spark 1.6.2 builds for Scala 2.10 even if you have Scala 2.11, and the Spark-Riak-Connector, by default, builds for Scala 2.10 as well. Therefore, in order for the streaming examples to work, the Scala version [config.sh](./src/main/repl/conf/config.sh) picks up must be the same as the Scala version that Spark and the Spark-Riak-Connector were built for. If you local Scala version is different than the version that Spark and the Spark-Riak-Connector were built for, you should change the Scala version in [this line](src/main/repl/conf/config.sh#L53) in [config.sh](./src/main/repl/conf/config.sh) to the version of Scala that Spark and the Spark-Riak-Connector were built for.
```

Comment:
Looks good. A few minor suggested changes:
- Change "for" to "with" in "...if your local Spark and the Spark-Riak-Connector were built for a different Scala version"
- Change "For example, Spark 1.6.2 builds for Scala 2.10 even if you have Scala 2.11, and the Spark-Riak-Connector, by default, builds for Scala 2.10 as well." to "For example, Spark 1.6.2 is built with Scala 2.10 by default."
- Typo in "If **you** local Scala version is different" 
- In last sentence, change occurrences of "built for" to "built with"